alg/sha256.c

```diff
@@ -392,7 +393,7 @@ SHA256_Final(uint8_t digest[32], SHA256_CTX * ctx)
 	insecure_memzero(ctx, sizeof(SHA256_CTX));
 
 	/* Clean the stack. */
-	insecure_memzero(tmp32, 288);
+	insecure_memzero(tmp32, sizeof(uint32_t) * 72);
```

Comment:
I should probably automatically recognize that 288 is `sizeof(uint32_t) * 72`, but it's not immediately obvious to me, so more than once I've panicked when browsing this code.  I eventually remember out that 32/8 = 4, but I think that `sizeof()` makes the code a lot more clear.